### PR TITLE
Add papers and edit README

### PR DESCRIPTION
**Pull request by nwlandry**
This addresses #419, #420, #421, and #424.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-# XGI
+### XGI ###
```
Comment (maximelucas) [inline README.md:1]: what's the difference with the previous format?

File: docs/source/about.rst [hunk 2]
```diff
@@ -5,13 +5,33 @@
...
+XGI is implemented in pure Python and is designed to seamlessly interoperate with the
+rest of the Python scientific stack (numpy, scipy, pandas, matplotlib, etc).  XGI is
+designed and developed by network scientists with the needs of network scientists in
+mind.
 
 - Repository: https://github.com/xgi-org/xgi
 - PyPI: `latest release <https://pypi.org/project/xgi/>`_
 - Twitter: `@xginets <https://twitter.com/xginets>`_
 - Contributors: `list <contributors.html>`_
+- Projects using XGI: `list <using-xgi.html`_
```
Comment (maximelucas) [inline docs/source/about.rst:31]:
```suggestion
- Projects using XGI: `list <using-xgi.html>`_
```

File: docs/source/index.rst [hunk 3]
```diff
@@ -70,6 +70,7 @@ mind.
 - PyPI: `latest release <https://pypi.org/project/xgi/>`_
 - Twitter: `@xginets <https://twitter.com/xginets>`_
 - Contributors: `list <contributors.html>`_
+- Projects using XGI: `list <using-xgi.html`_
```
Comment (maximelucas) [inline docs/source/index.rst:73]:
```suggestion
- Projects using XGI: `list <using-xgi.html>`_
```

File: docs/source/using-xgi.rst [hunk 4]
```diff
@@ -10,20 +10,40 @@ Published work
 2023
 ----
 
-- Yuanzhao Zhang, Maxime Lucas, and Federico Battiston, `"Higher-order interactions shape collective dynamics differently in hypergraphs and simplicial complexes" <http://doi.org/10.1038/s41467-023-37190-9>`_, Nature Communications **14**, 1605 (2023).
-- Nicolo Ruggieri, Martina Contisciani, Federico Battiston, and Caterina de Bacco, `"Community detection in large hypergraphs" <https://doi.org/10.1126/sciadv.adg9159>`_, Science Advances **9**, eadg9159 (2023).
+Yuanzhao Zhang, Maxime Lucas, and Federico Battiston, "Higher-order interactions shape collective dynamics differently in hypergraphs and simplicial complexes", Nature Communications **14**, 1605 (2023).
```
Comment (maximelucas) [inline docs/source/using-xgi.rst:13]:
```suggestion
Yuanzhao Zhang, Maxime Lucas, and Federico Battiston, "Higher-order interactions shape collective dynamics differently in hypergraphs and simplicial complexes", *Nature Communications* **14**, 1605 (2023).
```

File: docs/source/using-xgi.rst [hunk 5]
```diff
@@ -10,20 +10,40 @@ Published work
...
+
+:bdg-link-primary-line:`Paper <http://doi.org/10.1038/s41467-023-37190-9>`
+:bdg-link-primary-line:`Github <https://github.com/maximelucas/HOI_shape_sync_differently>`
+
+Nicolo Ruggieri, Martina Contisciani, Federico Battiston, and Caterina de Bacco, "Community detection in large hypergraphs", Science Advances **9**, eadg9159 (2023).
```
Comment (maximelucas) [inline docs/source/using-xgi.rst:18]:
```suggestion
Nicolo Ruggieri, Martina Contisciani, Federico Battiston, and Caterina de Bacco, "Community detection in large hypergraphs", *Science Advances* **9**, eadg9159 (2023).
```

File: docs/source/using-xgi.rst [hunk 6]
```diff
@@ -10,20 +10,40 @@ Published work
...
+
+:bdg-link-primary-line:`Paper <http://arxiv.org/abs/2305.06910>`
+:bdg-link-primary-line:`Github <https://github.com/nwlandry/filtering-higher-order-datasets>`
+
+Nicholas W. Landry and Juan Restrepo, "Polarization in hypergraphs with community structure", arXiv:2302.13967 (2023).
+
+:bdg-link-primary-line:`Paper <http://arxiv.org/abs/2302.13967>`
+:bdg-link-primary-line:`Github <https://github.com/nwlandry/filtering-higher-order-datasets>`
+
+Iacopo Iacopini, Márton Karsai, and Alain Barrat, "The temporal dynamics of group interactions in higher-order social networks", arXiv:2306.09967.
```
Comment (maximelucas) [inline docs/source/using-xgi.rst:39]:
```suggestion
Iacopo Iacopini, Márton Karsai, and Alain Barrat, "The temporal dynamics of group interactions in higher-order social networks", arXiv:2306.09967 (2023).
```

File: docs/source/using-xgi.rst [hunk 7]
```diff
@@ -10,20 +10,40 @@ Published work
...
+
+:bdg-link-primary-line:`Paper <https://arxiv.org/abs/2306.09967>`
+
+Timothy LaRock and Renaud Lambiotte, "Encapsulation Structure and Dynamics in Hypergraphs", arXiv:2307.04613.
```
Comment (maximelucas) [inline docs/source/using-xgi.rst:43]:
```suggestion
Timothy LaRock and Renaud Lambiotte, "Encapsulation Structure and Dynamics in Hypergraphs", arXiv:2307.04613 (2023).
```

**Review by maximelucas — APPROVED** [review]

**Comment by maximelucas** [comment]
Thanks Nich! A few details, otherwise good to go.

File: README.md (hunk 1 above)
Comment (nwlandry) [reply README.md:1]: The difference is that using the single hashtag is a title, whereas the triple hashtags on either side format it the same but exclude it from the table of contents.

File: docs/source/index.rst (hunk 3 above)
Comment (nwlandry) [reply docs/source/index.rst:73]: Good catch!!

File: docs/source/using-xgi.rst [hunk 8]
```diff
@@ -10,20 +10,40 @@ Published work
...
 ----
 
-- Yuanzhao Zhang, Maxime Lucas, and Federico Battiston, `"Higher-order interactions shape collective dynamics differently in hypergraphs and simplicial complexes" <http://doi.org/10.1038/s41467-023-37190-9>`_, Nature Communications **14**, 1605 (2023).
-- Nicolo Ruggieri, Martina Contisciani, Federico Battiston, and Caterina de Bacco, `"Community detection in large hypergraphs" <https://doi.org/10.1126/sciadv.adg9159>`_, Science Advances **9**, eadg9159 (2023).
+Yuanzhao Zhang, Maxime Lucas, and Federico Battiston, "Higher-order interactions shape collective dynamics differently in hypergraphs and simplicial complexes", *Nature Communications* **14**, 1605 (2023).
+
+:bdg-link-primary-line:`Paper <http://doi.org/10.1038/s41467-023-37190-9>`
+:bdg-link-primary-line:`Github <https://github.com/maximelucas/HOI_shape_sync_differently>`
+
+Nicolo Ruggieri, Martina Contisciani, Federico Battiston, and Caterina de Bacco, "Community detection in large hypergraphs", *Science Advances* **9**, eadg9159 (2023).
```
Comment (thomasrobiglio) [inline docs/source/using-xgi.rst:18]: I might be doing the greatest nitpick in xgi's history, the rest looks very cool, thank you @nwlandry  

```suggestion
Nicolò Ruggieri, Martina Contisciani, Federico Battiston, and Caterina de Bacco, "Community detection in large hypergraphs", *Science Advances* **9**, eadg9159 (2023).
```

Comment (nwlandry) [reply docs/source/using-xgi.rst:18]: No, that is actually super helpful, thanks so much!! I must have made an error when transcribing :)

**Pull request merged**